### ISSUE TITLE
Create secondary rounded button blue

### ISSUE THREAD

**Issue by RavindraP04** [Clueless-Community/seamless-ui#1610]
### Here is a reference to the component. 
![image](https://user-images.githubusercontent.com/98400348/216667819-ed0ee1c3-ea43-453e-9b76-5cb7bcdcdc51.png)

---
### After building the component please raise a PR with a screenshot of the component and add the component in `Button/src/Secondary/text-button-rounded.html` by replacing the existing code &  rename the file to `text-button-rounded-blue.html`. 

So, In short you need to do:-
- Make the Component
- Replace the code of `Button/src/Secondary/text-button-rounded.html` with your code
- Rename `text-button-rounded.html` to `text-button-rounded-blue.html`
  
If you need to use any icon please use it from [Hero Icons](https://heroicons.com/)

Good luck.

**Comment by Tapo41** [comment]
I want to contribute 
Please assign it to me. @RavindraP04

**Comment by SakshiShah29** [comment]
I would like to contribute.

**Comment by Tapo41** [comment]
#1749 @RavindraP04

**Comment by Tapo41** [comment]
#1749 I did this mistakenly ,u can delete that
